File: _src/06/Chapter6.VB/Maps/PersonViewInfoMap.vb
Imports System.Data.Entity.ModelConfiguration
Imports Chapter6.VB.Models

Namespace Maps
Public Class PersonViewInfoMap
    Inherits EntityTypeConfiguration(Of PersonViewInfo)
    Public Sub New()
        HasKey(Function(p) p.PersonId)
        ToTable("PersonView")
    End Sub
End Class
End Namespace

File: _src/06/Chapter6.CSharp/Maps/PersonViewInfoMap.cs
using System.Data.Entity.ModelConfiguration;
using Chapter6.CSharp.Models;

namespace Chapter6.CSharp.Maps
{
    public class PersonViewInfoMap :
        EntityTypeConfiguration<PersonViewInfo>
    {
        public PersonViewInfoMap()
        {
            HasKey(p => p.PersonId);
            ToTable("PersonView");
        }
    }
}
